### ISSUE TITLE
Any plans to test new BIOS to support zen5?

### ISSUE THREAD

**Issue by wesleywu** [SchmockLord/Hackintosh-ASRock-B650M-Riptide-7800X3D#4]
Thanks for sharing such detailed configuration and EFI

I plan to buy a zen5 9700X CPU for a hackintosh.

**Comment by SchmockLord** [comment]
Welcome. I am already on BIOS 3.06 Beta which runs well and is supporting 9000 Series. I don't see an issue buying an 9700X. Should work.

**Comment by zalech** [comment]
Just to mention: I haven't found 3.06 Beta on the asrock site and updated to 3.06, now I get reboot instead of shutdown. Used to use 1.30.AS05 [Beta].

upd: I'm on riptide btw.

**Comment by SchmockLord** [comment]
@zalech you are right. I just noticed. This restart instead of shutdown issue also was the reason why I stayed on 1.30 Beta 05 so long. Will go back to it now. Thanks for the hint.

**Comment by wesleywu** [comment]
so the BIOS version supporting zen5, like v3.06, will cause rebooting problem when shutting down?

**Comment by SchmockLord** [comment]
Yes. Can confirm that.

**Comment by Messa1** [comment]
@SchmockLord any update on bios versions that should work now?

**Comment by SchmockLord** [comment]
@Messa1 I have a new version of my EFI which also works with 3.10 and doesn't have this restart instead of shutdown. Also sleeps works fine, even in Sequoia. The trick is, to disable the XHC Ports, so XHC0, XHC1 and XHC2. They still work but shown as "AppleUSBXHCI" in Hackintool, not with the XHC0... naming.
I will later update the repo.

**Comment by SchmockLord** [comment]
@Messa1 Try this: https://github.com/SchmockLord/Hackintosh-ASRock-B650M-Riptide-7800X3D/releases/tag/v19
